src/matrix/room/BaseRoom.js

```diff
@@ -378,6 +378,10 @@ export class BaseRoom extends EventEmitter {
         return this._summary.data.membership;
     }
 
+    get powerLevels() {
+        return this._timeline.powerLevels;
```

Comment:
`this._timeline` is only set if the timeline is open, which will be fine for using it in the in the member list, but it will throw if `openTimeline` is not called before `loadMemberlist`. Ideally we'd:

 - lazy load the power levels in BaseRoom either when loading the members or opening the timeline, and pass them into Timeline and Memberlist.
 - if loaded, update the power levels in afterSync of Room
 - resort the member list when the power levels change in afterSync.